lib/smooth/application/network/http/regular/MIMEParser.cpp

```diff
@@ -112,6 +150,176 @@ namespace smooth::application::network::http::regular
         return b;
     }
 
+    void MIMEParser::myparse(const uint8_t* p, std::size_t length, const myFormDataCallback& content_callback,
+                             const URLEncodedDataCallback& url_data, const uint16_t chunksize)
+    {
+        static enum class Status
+        {
+            Begin,
+            Headers,
+            Data,
+            End
+        }
+        status = Status::Begin;
+        static std::string id{};
+        static std::string filename{};
+        BoundaryIterator begin{};
+        BoundaryIterator end{};
+        static bool first_part = false;
+        static bool end_of_transmission = false;
```

Comment:
These static, can they? If there are multiple instances of the MIMEParser (as will happen if you have multiple http-server on different threads), then they'd overwrite these static variables. They need to be made into class-members instead.

Reply:
o.k. fixed